modules/auxiliary/scanner/http/dlink_dir_session_cgi_http_login.rb

```diff
@@ -42,6 +42,8 @@ def initialize
         OptPath.new('PASS_FILE',  [ false, "File containing passwords, one per line",
```

Comment:
had a hard time getting this exploit to work before these changes, so I suspect it is mildly broken